cypress/integration/firm_deputy_hub.spec.js

```diff
@@ -11,32 +11,62 @@ describe("Firm Deputy Hub", () => {
             cy.contains(".moj-header__link", "Sirius");
         });
 
-        const expected = [
-            "Supervision",
-            "LPA",
-            "Admin",
-            "Logout",
-        ];
+        const expected = ["Supervision", "LPA", "Admin", "Logout"];
 
         it("has working nav links within header banner", () => {
             cy.get(".moj-header__navigation-list")
                 .children()
                 .each(($el, index) => {
                     cy.wrap($el).should("contain", expected[index]);
-                    let $linkName = (expected[index].toLowerCase());
-                    cy.wrap($el).find('a').should("have.attr", "href").and("contain", `/${$linkName}`);
-
+                    let $linkName = expected[index].toLowerCase();
```

Comment:
No need to change it as it is inconsequential but only use let if you are planning on changing its value. Prefer immutability where possible.